spec/fixtures/missing_partials.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I think this belongs in the next commit(?)

Reply:
OOPS!!!!

Reply:
Fixed